temp/.gitignore

```diff
@@ -1,2 +0,0 @@
-**
```

Comment:
Oops!  I didn't mean to remove this.  I had thought it was a file I had accidentally added.

Reply:
I put the file back :)